packages/stitch/src/types.ts

```diff
@@ -68,6 +68,7 @@ export interface IStitchSchemasOptions<TContext = any> extends Omit<IExecutableS
   mergeDirectives?: boolean;
```

Comment:
@yaacovCR On L63 – https://github.com/gmac/graphql-tools/blob/master/packages/stitch/src/types.ts#L63

```graphql
subschemas?: Array<GraphQLSchema | SubschemaConfig | Array<SubschemaConfig>>;
```

Is the array of arrays still supported? The shared endpoint seems simpler to me.

Reply:
still supported, we can phase out in v7 if we want